src/components/OptionsRow.js

```diff
@@ -1,124 +1,360 @@
+//             <span className="description"> Total Cloud Fraction</span>{" "}
+//           </>
+//         ),
+//       },
+//       {
+//         value: "hurs",
+//         label: (
+//           <>
+//             {" "}
+//             <span className="tag other">hurs</span>{" "}
+//             <span className="description"> Near-Surface Relative humidity</span>{" "}
+//           </>
+//         ),
+//       },
+//       {
+//         value: "psl",
+//         label: (
+//           <>
+//             {" "}
+//             <span className="tag other">psl</span>{" "}
+//             <span className="description"> Sea Level Pressure</span>{" "}
+//           </>
+//         ),
+//       },
+//       {
+//         value: "rss",
+//         label: (
+//           <>
+//             {" "}
+//             <span className="tag other">rss</span>{" "}
+//             <span className="description"> Net surface long wave flux</span>{" "}
+//           </>
+//         ),
+//       },
+//       {
+//         value: "rls",
+//         label: (
+//           <>
+//             {" "}
+//             <span className="tag other">rls</span>{" "}
+//             <span className="description"> Net surface long wave flux</span>{" "}
+//           </>
+//         ),
+//       },
+//     ],
+//   },
+// ];
 
 // TODO: This list will be updated as more plots are generated. It covers ~100
 // years.
-const periodOptions = [
-  { value: '19890101-20081231', label: '1989-2008' }
-]
+// const periodOptions = [{ value: "19890101-20081231", label: "1989-2008" }];
 
 // TODO: Check season names are OK with research team as typical users might be
 // more familiar with value names e.g. DJF
-const seasonOptions = [
-  { value: 'DJF', label: 'DJF - Winter' },
-  { value: 'MAM', label: 'MAM - Spring' },
-  { value: 'JJA', label: 'JJA - Summer' },
-  { value: 'SON', label: 'SON - Autumn' }
-]
+// const seasonOptions = [
+//   { value: "DJF", label: "DJF - Winter" },
+//   { value: "MAM", label: "MAM - Spring" },
+//   { value: "JJA", label: "JJA - Summer" },
+//   { value: "SON", label: "SON - Autumn" },
+// ];
 
 // TODO: More plot types to be added.
-const plottypeOptions = [
-  { value: 'eval-boxplots', label: 'Box plots of spread within each ensemble' },
-  { value: 'eval-maps', label: 'Maps of observations & biases' },
-  { value: 'eval-pca-gcm', label: 'PCA GCM plots' },
-  { value: 'eval-pca-rcm', label: 'PCA RCM plots' },
-  { value: 'eval-Taylor', label: 'Taylor plots' }
-]
+// const plottypeOptions = [
+//   { value: "eval-boxplots", label: "Box plots of spread within each ensemble" },
+//   { value: "eval-maps", label: "Maps of observations & biases" },
+//   { value: "eval-pca-gcm", label: "PCA GCM plots" },
+//   { value: "eval-pca-rcm", label: "PCA RCM plots" },
+//   { value: "eval-Taylor", label: "Taylor plots" },
+// ];
+
+// TODO: actual values to be added.
+// const regionOptions = [{ value: "region value", label: "region" }];
```

Comment:
I'd get rid of everything commented out here again to clean the file a bit. We won't lose the information as it's in previous commits we can revert to easily.